### DISCUSSION TITLE
[BUG] Pending claw back transaction

### DISCUSSION THREAD

**Discussion by JtabaresH** [Chia-Network/chia-blockchain#15617] [Support]
Good afternoon.
I have a problem and I would like to know if someone can help me.
I was testing the new clawback function and I made a transaction that got stuck in "clawback" and I haven't been able to recover the balance of my wallet for 10 days because the txn is still stuck, already resynchronized several times and suspects the basic fee of this new function, but nothing seems to work and i am already desperate to have my balance stuck.
Thanks in advance and any guidance is appreciated.

**Comment by jonathanong** [comment]
Same issue here

**Reply by technologiespro** [reply]
solution:

get ids transactions:
>> `chia wallet get_transactions`

/**
Transaction fff...aaa
Status: Pending
Amount received in clawback as sender: 0.1 XCH
**/

refund:
>> `chia wallet clawback -m 0.0000005 -ids fff..aaa --force --push`

{'success': True, 'transaction_ids'...

**Comment by BrandtH22** [comment]
Hey @JtabaresH and @jonathanong , There are a couple of known bugs regarding clawbacks.
The first fix for these has been included in 2.0.1 (and 2.0.0) so I would first recommend upgrading to 2.0.1 and reattempting the clawback.

Note that this might not work if the issue is related to the second bug, currently there is no fix for the second bug but there is a workaround in place in the latest dev version and on the main branch in the chia github repo.

If you are interested in attempting the workaround I recommend jumping into our discord server (https://discord.gg/chia) where we can provide more thorough support during the process of upgrading to the latest dev or main branch, using the workaround, and downgrading back to the current version.

If you want to try it on your own here are the steps (NOTE - git is required for these steps):
1. Backup your keys file just for safety (should not effect anything but always better to be safe)
2. Close chia
3. in powershell (windows) or terminal (linux) run `git clone https://github.com/Chia-Network/chia-blockchain.git -b main --recurse-submodules`
4. run `cd chia-blockchain`
5. windows: run `.\install.ps1` | linux: run `sh install.sh`
6. windows: run `.\venv\scripts\Activate.ps1` | linux: run `. ./activate`
7. run `chia start wallet`
8. Identify the clawback transaction id by running `chia wallet get_transactions --clawback`
9. Use the transaction id in this command and for windows run `chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'` | for linux run `chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`
10. Monitor a blockchain explorer, use the command provided in the previous output, or use `chia wallet show` to monitor for when the clawback has been received by your wallet
11. run `chia stop all -d`
12. delete the "chia-blockchain" folder and start chia as you normally would have previously

**Reply by koblevo** [reply]
Hi guys,

Has the issue been resolved as I have the same problem? I tried to use the claw back option on Chia GUI Client 1.8.2 - created the transaction from my wallet to OKX exchange and my transaction got stuck in Pending status for 3 days now... I upgraded the GUI client to version 2.0.1, but no option to claim the transaction back. Please advise if BrandtH22 steps worked for you. Thanks in advance for any reply.
![Screenshot 2023-09-17 214546](https://github.com/Chia-Network/chia-blockchain/assets/81450978/41ef98b8-9541-4e56-b4cf-98ed1006d8d9)

**Reply by shaneo257** [reply]
looks like you may have the 2nd bug brandt was talking about, you can download the latest dev version from here https://downloads.chia.net/latest-dev/ and try the above steps from step 7

**Reply by koblevo** [reply]
Thanks Shaneo257 for the reply.

**Comment by BrandtH22** [comment]
Hey @JtabaresH @jonathanong @koblevo , were all of you able to force the clawback transaction to go through with the provided instructions?

**Comment by koblevo** [comment]
Hi Support Team,

Thank you for your response.

I have already resolved the issue: updated the wallet to the latest version
and recynced it.

Kind Regards,
Yuriy

**Comment by mgbanii** [comment]
I have the same issue on latest version and non of above solutions work

**Reply by mgbanii** [reply]
anyone there?

**Reply by mgbanii** [reply]
![Untitled](https://github.com/Chia-Network/chia-blockchain/assets/35780142/c362a23a-8c35-468c-9041-5e6c5165ec38)

**Reply by BrandtH22** [reply]
What I would do is:
1. Delete unconfirmed transactions from the GUI
2. Go through the above process for installing the newest version and forcing the clawback

Note - we can generally provide more timely and thorough support in our discord (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
